operator/handler/schedule_test.go

```diff
@@ -184,6 +184,33 @@ func TestScheduleHandler_mergeBackendWithDefaults(t *testing.T) {
 	}
 }
 
+func TestScheduleHandler_mergePodSecurityContextWithDefaults(t *testing.T) {
+	var nilPodSecurityContextPointer *corev1.PodSecurityContext
+	tests := map[string]struct {
+		givenSchedulePodSecurityContext *corev1.PodSecurityContext
+		givenResourcePodSecurityContext *corev1.PodSecurityContext
+		expectedPodSecurityContext      *corev1.PodSecurityContext
+	}{
+		"Given_ResourcePSC_When_SpecPSC_Is_Nil_Then_UseResourcePSC": {
+			givenSchedulePodSecurityContext: nilPodSecurityContextPointer,
+			givenResourcePodSecurityContext: newPodSecurityContext(0, 1, corev1.SELinuxOptions{Level: "s0:c123,c456"}),
+			expectedPodSecurityContext:      newPodSecurityContext(0, 1, corev1.SELinuxOptions{Level: "s0:c123,c456"}),
+		},
```

Comment:
If you've already prepare test cases, please add cases that verifies the existing behaviour